README.md

```diff
@@ -228,6 +228,15 @@ $index->search('prince', ['limit' => 1])
 }
 ```
 
+With limit and filter, both single and double quotes are supported.
+```php
+// Enclosing with double quotes
+$index->search('prince', ['limit' => 1, 'filter' => "title = 'Le Petit Prince' OR book_id = 456"]);
```

Comment:
```suggestion
$index->search('prince', ['limit' => 1, 'filters' => "title = 'Le Petit Prince' OR book_id = 456"]);
```

Reply:
oops, not sure how that end up there lol - thanks